packages/chrome/src/elements/Chrome.tsx

```diff
@@ -5,12 +5,55 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import React, { HTMLAttributes } from 'react';
+import React, { HTMLAttributes, useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { ThemeProps, DefaultTheme } from 'styled-components';
+import readableColor from 'polished/lib/color/readableColor';
+import { withTheme, DEFAULT_THEME, getColor } from '@zendeskgarden/react-theming';
+import { ChromeContext } from '../utils/useChromeContext';
 import { StyledChrome } from '../styled';
 
+interface IChromeProps extends HTMLAttributes<HTMLDivElement> {
```

Comment:
This context only affects the `Nav` item, right? Could we move this prop and the context provider to the `Nav` component to limit its scope?

Reply:
No, it affects `SubNav` (and any future Chrome-related components that should be affected by the top-level hue).